sormas-ui/src/main/java/de/symeda/sormas/ui/utils/CommitDiscardWrapperComponent.java

```diff
@@ -423,6 +430,19 @@ public void run() {
 		return deleteButton;
 	}
 
+	public Button getDeleteWithReasonButton(String entityName) {
+		if (deleteButton == null) {
+
```

Comment:
Remove unnecessary newlines in this method

Reply:
removed